pkg/kube/apisix/apis/config/v1/types.go

```diff
@@ -83,7 +84,8 @@ type ApisixUpstream struct {
 	metav1.TypeMeta   `json:",inline" yaml:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
 
-	Spec *ApisixUpstreamSpec `json:"spec,omitempty" yaml:"spec,omitempty"`
+	Spec   *ApisixUpstreamSpec   `json:"spec,omitempty" yaml:"spec,omitempty"`
```

Comment:
I'm wondering whether we can just use value for `Spec` field, not pointer.

Reply:
This can be modified in another PR.

Reply:
Got it.